sdk/messaging/azeventhubs/processor_test.go

```diff
@@ -529,8 +539,13 @@ func testWithLoadBalancer(t *testing.T, loadBalancerStrategy azeventhubs.Process
 	// Create the checkpoint store
 	// NOTE: the container must exist before the checkpoint store can be used.
 	t.Logf("Checkpoint store created")
-	checkpointStore, err := checkpoints.NewBlobStore(cc, nil)
-	require.NoError(t, err)
+	var checkpointStore *countingCheckpointStore
+	{
+		inner, err := checkpoints.NewBlobStore(cc, nil)
+		require.NoError(t, err)
+
+		checkpointStore = &countingCheckpointStore{BlobStore: inner, listed: 0}
+	}
```

Comment:
TIL Go allows anonymous blocks 🤯

Reply:
Yeah, they don't work for defer's - those still go to the function level, but you can scope variables, etc..